addons/account/data/security_mail_templates.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <template id="partner_bank_account_changed_template" >
+            <p>Dear <t t-esc="user_name" />,<br/><br/>
+                You receive this email because we would like to be sure that you are the person who changed the bank account 
```

Comment:
I don't understand your comment, this template is called twice : when a bank account is added/removed and when a bank account is edited. Is that what you want ?

Reply:
yes.. but the text in the template doesn't say so.
```suggestion
                You receive this email because we would like to be sure that you are the person who changed the bank account or created one, or removed one. 
```
this is ugly but you get the idea